ui/app/src/components/LauncherSiteCard/LauncherSiteCard.tsx

```diff
@@ -73,25 +77,33 @@ function LauncherSiteCard(props: LauncherSiteCardProps) {
 
   return (
     <>
-      <Box
-        // @ts-ignore
-        button
+      <ListItemButton
+        disabled={!isSiteReady}
         selected={selected}
-        boxShadow={1}
         component={ListItem}
-        onClick={() => onCardClick(value)}
+        onClick={() => isSiteReady && onCardClick(value)}
         className={cx(classes.card, props.classes?.root)}
+        sx={{ position: 'relative' }}
         title={title}
       >
-        <ListItemText primary={title} primaryTypographyProps={{ className: classes.siteName, noWrap: true }} />
-        {hasOptions && (
-          <ListItemSecondaryAction>
-            <IconButton aria-label="settings" onClick={handleOptions} size="large">
-              <MoreVertIcon />
-            </IconButton>
-          </ListItemSecondaryAction>
-        )}
-      </Box>
+        <ListItemText
+          primary={title}
+          primaryTypographyProps={{ className: classes.siteName, noWrap: true }}
+          sx={{ paddingRight: isSiteReady ? 0 : '35px' }}
+        />
+
+        <ListItemSecondaryAction>
+          {isSiteReady ? (
+            hasOptions && (
+              <IconButton aria-label="settings" onClick={handleOptions} size="large">
+                <MoreVertIcon />
+              </IconButton>
+            )
+          ) : (
+            <SiteStatusIndicator state={state} />
+          )}
+        </ListItemSecondaryAction>
```

Comment:
Why are you keeping the `ListItemSecondaryAction` even when nothing is being rendered on it? Is it intentional? Look back and my original code suggestion on previous review.